test/ws-prefix-rewrite-core.js

```diff
@@ -11,7 +11,7 @@ const got = require('got')
 
 const level = 'warn'
 
-async function proxyServer (t, backendURL, backendPath, proxyOptions, wrapperOptions) {
+async function proxyServer(t, backendURL, backendPath, proxyOptions, wrapperOptions) {
```

Comment:
can we avoid these formatting changes? they're inconsistent with the rest of the codebase, e.g. [here](https://github.com/fastify/fastify-http-proxy/pull/172/files#diff-269802b67ef10bddd18881f6f23f2f9555974a2ccfed0ead1ef4eb79ce889af7R34). Either we change it everywhere or we don't change it. I would go with the latter.